content/_data/changelogs/weekly.yml

```diff
@@ -5291,6 +5291,42 @@
     authors:
       - jglick
 
+- version: '2.204'
+  date: 2019-11-10
+  changes:
+
+  # RFEs
+
+  - type: rfe
+    pull: 4273
+    issue: 59679
+    message: |-
+      Plugin manager descriptions always link to the plugins site instead of the Jenkins wiki.
+  - type: rfe
+    pull: 4341
+    message: |-
+      Increase the number of datapoints recorded for multistage time series graphs such as those used for load statistics to display data for 6 hours, 48 hours, and 56 days (up from 1 hour, 24 hours, and 28 days).
+  - type: rfe
+    pull: 4340
+    references:
+      - url: https://github.com/jenkinsci/remoting/releases/tag/remoting-3.36
+        title: Remoting release notes
+    message: |-
+      Update Remoting from 3.35 to 3.36 to add new command line options "-help" and "-version".
+
+  # Developer-facing changes (APIs, extensions, etc.)
+
+  - type: bug
+    pull: 4311
+    issue: 59886
+    message: |-
+      Developer: Strengthen the queue to avoid canTake and canRun implementers hanging the queue
```

Comment:
See https://github.com/jenkinsci/jenkins/releases/tag/jenkins-2.204 , current changelog is not clear without opening the PR

Reply:
Resolved in 294b156969d115c1782fc9c9fc0ba60be7a3bed3 and b698094496745ba685590521d9dbd00f9e680faa